docs/manual/hybrid_apps.rst

```diff
@@ -0,0 +1,47 @@
+Hybrid Apps
+===========
+
+.. note::
+
+    Check out the `Hybrid App Sample Project <https://github.com/GodotVR/godot_openxr_vendors/tree/master/samples/hybrid-app-sample>`_
+    for a working demo of hybird app functionality.
+
+Hybrid apps are applications capable of swapping between an immersive 3D mode and a 2D panel mode at runtime.
+
+Project Settings
+----------------
+
+To enable hybrid app functionality, the feature must be enabled in your project settings. The setting can be found in **Project Settings** under the **OpenXR** section.
+The **Hybrid App** setting should be visible if **Advanced Settings** are enabled.
+
+.. image:: img/hybrid_apps/hybrid_apps_project_setting.png
+
+By default, hybrid app functionality is disabled. Selecting either of the other options will enable hybrid app features and determine the app's starting mode.
+
+Switching Modes
+---------------
+
+To switch between immersive and panel app modes in your hybrid app, use the :ref:`switch_mode <class_openxrhybridapp_method_switch_mode>` function.
+In order to pass data between modes, this function accepts a ``String`` that can be accessed in the new app mode via the :ref:`get_launch_data <class_openxrhybridapp_method_get_launch_data>` function.
+Storing the data you'd like to share in a `Dictionary <https://docs.godotengine.org/en/stable/classes/class_dictionary.html>`_
+and converting it to/from `JSON <https://docs.godotengine.org/en/stable/classes/class_json.html>`_ is one way you might utilize this feature.
+
+Here's what it might look like if your app is switching from immersive to panel mode and you have a value you'd like to pass along:
+
+.. code-block:: gdscript
+
+    var my_value = 100 # Value we want to share.
+
+    var my_dictionary = {}
+    my_dictionary["my_key"] = my_value
```

Comment:
Maybe use variable names with more context to help folks understand what this is for?

Like:

```suggestion
    var current_score = 100 # Game state we want to keep in the new mode.

    var my_dictionary = {}
    my_dictionary["current_score"] = current_score
```